lib/verifier/catalog_index_verifier.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require "faraday"
+require_relative "../verifier"
+require_relative "../derivatives"
+
+# Verifies that catalog indexing workflow stage did what it was supposed to.
+
+module PostZephirProcessing
+  class CatalogIndexVerifier < Verifier
```

Comment:
Should we consider e.g. `Verifier::CatalogIndex` as we are doing with `Derivative`?

Reply:
Future work: https://github.com/hathitrust/post_zephir_processing/issues/55